### DISCUSSION TITLE
Highlight selected Airport types

### DISCUSSION THREAD

**Discussion by bcol80** [albar965/littlenavmap#1008] [Ideas]
Most buttons on the toolbar get highlighted when selected but the new Airports menu doesn't do this so you can't see at a glance which airports you have selected to view or not. Would be nice if there was some sort of highlighting here

![image](https://user-images.githubusercontent.com/87935689/224015013-85e00048-bbf8-4e80-90bf-57dcc4bb9e5b.png)

**Comment by albar965** [comment]
The button is highlighted if anything is different from default (show all) which is mostly the case (I have heliports and seaplane bases always off when planning). Not terribly useful, I admit.

Shown features are already noted in the status bar tooltip but this is confusing due to the mass of information.

Possible solutions: 
1. Show small tags at the icon to indicate settings. Probably not enough space and then barely visible.
2. Show settings in tooltip. Only visible if hovering the mouse above.

I'm not sure how to improve this.

**Reply by bcol80** [reply]
I never actually realised it showed this on the statusbar. That's pretty handy. 

I thought of another solution as well. The map pulldown menu (with open street map etc) has little dots next to the map that's selected and which projection is in use. Could small dots be added next to the selected airport types as well? Or is it possible to make a change to the small airfield icon types to indicate if they're selected or not? Like a shaded background or something?

**Reply by albar965** [reply]
> I never actually realised it showed this on the statusbar. That's pretty handy.
> 
> I thought of another solution as well. The map pulldown menu (with open street map etc) has little dots next to the map that's selected and which projection is in use. Could small dots be added next to the selected airport types as well? Or is it possible to make a change to the small airfield icon types to indicate if they're selected or not? Like a shaded background or something?

The dots are the toolbar handle. You can click and move the toolbar with it. https://www.littlenavmap.org/manuals/littlenavmap/release/latest/en/INTRO.html#toolbars

I'll see what can be done. A problem is the small icon which does not allow to add tags.
